src/MSBuildLocator/MSBuildLocator.cs

```diff
@@ -139,14 +139,50 @@ public static void RegisterInstance(VisualStudioInstance instance)
         /// </param>
         public static void RegisterMSBuildPath(string msbuildPath)
         {
-            if (string.IsNullOrWhiteSpace(msbuildPath))
+            RegisterMSBuildPath(new string[] {
+                msbuildPath
+#if NET46 && !MONO
```

Comment:
Because mono runs "unmodified" .NET Framework binaries, it doesn't usually get a compile-time constant (and can't easily have a specific binary in a NuGet package). I think this check would need to happen at runtime and use `IsRunningOnMono` introduced in #82. Since that isn't landed yet, let's just drop it here and add a note to that PR to merge this and add the check?

Reply:
Sounds good to me.